File: java/oop/CityApp/src/com/techlabs/citymap/City.java
package com.techlabs.citymap;

public class City {
	private String cityCode;
	private String cityDescription;
	
	public City(String cityCode,String cityDescription){
		this.cityCode=cityCode;
		this.cityDescription=cityDescription;
	}
	
	public String getCityCode(){
		return this.cityCode;
	}
	
	public String getCityDescription(){
		return this.cityDescription;
	}
	
	@Override
	public String toString() {
		String string="CityCode: "+this.cityCode
				+"\nDescription: "+this.cityDescription;
		
		return string;
	}
}

File: dotNet/CSharp/CityApp/CityApp/City.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CityApp
{
    class City
    {
        private string _cityCode;
        private string _cityDescription;

        public City(string cityCode,string cityDescription)
        {
            this._cityCode = cityCode;
            this._cityDescription = cityDescription;
        }

        public String CityCode
        {
            get
            {
                return _cityCode;
            }
        }

        public String CityDescription
        {
            get
            {
                return _cityDescription;
            }
        }

        public override string ToString()
        {
            return "CityCode: " + this.CityCode
                +"\nCityDescription: "+this.CityDescription;
        }
    }
}
